packages/react/src/components/Table/TableBody/_table-dnd.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../globals/vars';
+@import '../../../globals/spacing';
+
+// The 6-dots grable handle area on a row that can be dragged. This is really that the parent of
+// that picture so the draggable area is a larger, square.
+.#{$iot-prefix}--table-drag-handle {
+  height: 2rem;
+  padding-inline: 1rem;
+  display: grid;
+  align-items: center;
+  justify-items: center;
+
+  cursor: grab;
+  user-select: none;
+
+  position: relative;
+  inset-inline-start: var(--row-nesting-offset);
+}
+
+.bx--data-table td.#{$iot-prefix}--table-grab-handle-cell {
+  // remove padding so that the handle can fill the whole cell. Visually it's smaller, but the
+  // draggable region will be large.
+  padding-inline: 0;
+}
+
+// When hovering over a row. Draws a border around the row using an extra overlay element positioned
+// over it since CSS alone can add the require border around the row.
+.#{$iot-prefix}--table-drop-row-overlay {
+  // Abs positioned, but JavaScript will set its rect
+  position: absolute;
+  border: dashed 2px $carbon--blue-60;
+  pointer-events: none;
+  // This probably doesn't need to be very high, as long as it's higher than the table.
+  z-index: 1000;
+}
+
+// The shadowed tile shown while dragging a row. The caller supplies the text or icon inside this
+// through a callback, otherwise it just a white square.
+.#{$iot-prefix}--table-drag-avatar {
+  font-size: 0.875rem; // to match table default
+  // Abs positioned, but JavaScript will set its rect
+  position: absolute;
+  top: 0;
+  left: 0;
+  border: solid 1px $carbon--blue-60;
+  // background-color: $carbon--gray-10;
+  background-color: white;
+  pointer-events: none;
+  padding: $spacing-05;
+  padding-inline-start: 2rem; // extra space at the start
+  // Above the row overlay
+  z-index: 1001;
```

Comment:
Same comment. Not sure if these are relative to the Table context or in their own but something to verify.